script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -346,17 +347,17 @@ Then while I was still in a state of shock, you put an end to my vital organs wi
 [sha:01c7d5d0d6769fc11f45f891eff0bbfcc56a8b1b]{
 -- Page 61, Offset 8742.
 -- どのくらい完璧だったかっていうと、あの時の志貴をそのままキャンバスに閉じ込めれば、それ以上はないほどの<芸術品|え>になるぐらい、完璧だった。
-If you were to ask me how perfect it was, I'd say it was so perfect that if you had captured that moment on canvas, it would have become a work of art unmatched by any other.
+If you were to ask me how perfect it was, I'd say it was so perfect that if you had captured that moment on a canvas, it would've become a work of art unmatched by any other.
```

Comment:
Not sure you need the `a` before canvas since it's a medium as well as an object